doc/README.md

```diff
@@ -1,4 +1,60 @@
 KOS_DOC
 =======
+The documents are generated using Sphinx restructured text, with the ReadTheDocs
+theme.
 
-New documentation source for Sphinx/reStructuredText format.
+#Getting started on Windows
+
+(For this example, the KOS repository is assumed to be located at `C:\KOS`,
+you should adjust your path based on your actual repository location)
+
+1. If you don't already have Python installed, install the latest version in the
+  2.7 series.  At the time of this writing, 2.7.11 was the most current version.
+2. You can verify the installation and version of Python (and pip) by issuing
+  the following commands from the command line:
+  ```
+  C:\>python --version
+  Python 2.7.11
+  C:\>pip --version
+  pip
+  ```
+3. Install the Sphinx engine
+  ```
+  C:\>pip install sphinx
+  ```
+4. Ensure that the read the docs template is installed.
+  ```
+  C:\>pip install sphinx_rtd_theme
+  ```
+5. Switch to the docs directory and run the make batch file:
+  ```
+  C:\>cd KOS\docs
+  C:\KOS\docs>make html
+  ```
+6. Review the output for errors and warnings.  In the above example you would
+  find the compiled html files at `C:\KOS\docs\gh-pages`
+7. (Optional) You may browse the generated html using file urls, or by using
+  Python's included SimpleHTTPServer:
+  ```
+  C:\KOS\docs>cd gh-pages
+  C:\KOS\docs\gh-pages>python -m SimpleHTTPServer 8000
+  Serving HTTP on 0.0.0.0 port 8000...
+  ```
+  At which point you can point your browser to `http://localhost:8000`
+
+#Getting started on Linux
+1. TODO: WRITE LINUX INSTRUCTIONS WITH APT-GET AND PIP
+
+#Publishing
+1. Clone KSP-KOS/KOS gh-pages branch if not done already, pull the most recent
+  version from the upstream repository if you already have a clone.
+2. The clone should either be in a different folder from the primary
+  repository or you need to copy the rendered `doc/gh-pages` contents to a
+  temporary folder
+3. Delete the contents of the repository root, except the `.git` folder,
+  `.gitatributes` file, and `.nojekyll` file.
```

Comment:
It's a little of both.  All dot files are presumed to not belong in the repo except where explicitly noted (``.gitignore`` is one exception we explicitly made).  That means both not adding dot files to the repo, and not assuming the repo will repopulate your dot files for you when you want to switch back to the normal branches and not work in gh-pages anymore.

It's the last one that I have a problem with in your instructions.  You're telling people to destroy their only copy of the dot file, under the assumption it will come back again when they switch branches back, and it won't because it's not a tracked file (actually, that same complaint extends to the entire way its being done here, where it's being done in the same location as the normal development, so all local untracked files have to be destroyed in the user's local working folder).

I strongly dislike the way github organized gh-pages for this reason.  It really should have put them in a separate subdirectory so you don't have to destroy all your local untracked files when you switch branches to it.

Here's how I deal with the gh-pages mess:  I make a totally separate secondary clone of the fork that I use ONLY for gh-pages, and then after the ``make html`` step, I do the copy from the main repo's gh-pages directory that ``make html`` created to this secondary repo's root folder, and perform the commit and push of gh-pages from the secondary one.  That way I don't need to clobber all my local untracked stuff.  I think that may make for simpler instructions, and make my comment about dot files moot.